scripts/src/models/index.ts

```diff
@@ -0,0 +1,47 @@
+import "reflect-metadata";
+import { BaseEntity, createConnection } from "typeorm";
+import { ConnectionOptions } from "typeorm/connection/ConnectionOptions";
+
+import { getConfig } from "../config";
+import { normalizeError, path } from "../utils";
+
+const dbConfig = Object.assign(getConfig().db);
+if (dbConfig.type === "sqlite" && !/^[./]/.test(dbConfig.database)) {
```

Comment:
what?

Reply:
Well, if using sqlite and the db file path isn't relative/absolute then make the path relative to the root directory of the project.
For example, take what's in our `config/default.json` configuration:
```
...
"database": "database.sqlite",
...
```
The file that will be used will be `PROJECT_ROOT/database.sqlite`